janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphBaseTest.java

```diff
@@ -66,12 +68,17 @@
     public StoreFeatures features;
     public JanusGraphTransaction tx;
     public JanusGraphManagement mgmt;
+    private static final JanusGraphManager graphManager;
 
     public Map<String,LogManager> logManagers;
 
     public JanusGraphBaseTest() {
     }
 
+    static {
+        graphManager = new JanusGraphManager(new Settings());
```

Comment:
I added the method signature because I think it is a good idea, but now looking back I am not sure there is a reason to actually call it from these tests since the tests instantiate their own `JansuGraphManager` anyway, so there is no worry of overlapping state.